### PR TITLE
[OTA-2701] User sees ODN equals 'MC's Organization

### PR DESCRIPTION
**Pull request by piotrcichonmee**
Signed-off-by: Piotr Cichon <ext-piotr.cichon@here.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/stores/UserStore.js [hunk 1]
```diff
@@ -42,10 +43,15 @@ export default class UserStore {
...
   }
 
-  getNamespaces = async () => {
-    const response = await axios.get(API_USER_NAMESPACES);
+  getOrganizations = async () => {
+    const response = await axios.get(API_USER_ORGANIZATIONS);
     const { data } = await response;
-    this.userNamespace = data.length ? data[0] : '';
+    this.userOrganizationName = data.find(organization => organization.landing).name;
+    if (!this.userOrganizationName) {
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/stores/UserStore.js]: @houcros are you doing this on BE too?

Comment (houcros) [reply ota-plus-web/app/reactapp/src/stores/UserStore.js]: Yes, this should be merged with https://github.com/advancedtelematic/ota-plus-user-profile/pull/84.

Comment (houcros) [reply ota-plus-web/app/reactapp/src/stores/UserStore.js]: (But then there would be no need to use the initials here.)

File: ota-plus-web/app/reactapp/src/stores/UserStore.js [hunk 2]
```diff
@@ -42,10 +43,15 @@ export default class UserStore {
     resetAsync(this.contractsAcceptAsync);
   }
 
-  getNamespaces = async () => {
-    const response = await axios.get(API_USER_NAMESPACES);
+  getOrganizations = async () => {
+    const response = await axios.get(API_USER_ORGANIZATIONS);
     const { data } = await response;
-    this.userNamespace = data.length ? data[0] : '';
+    this.userOrganizationName = data.find(organization => organization.landing).name;
```
Comment (houcros) [inline ota-plus-web/app/reactapp/src/stores/UserStore.js]: @piotrcichonmee this field was renamed from `landing` to `isCreator`.

**Pull request merged**